README.md

```diff
@@ -1,6 +1,11 @@
 # Jupyterlab App
 
-# test
-
-A native app for JupyterLab, based on electron.
+A native app for [JupyterLab](https://github.com/jupyterlab/jupyterlab), based on electron.
 Project for Cal Poly interns of summer 2017. 
```

Comment:
Can you just remove this line? It was added as a test so we could practice doing pull requests, and it's unnecessary/unprofessional.